include/envoy/grpc/async_client.h

```diff
@@ -91,6 +104,20 @@ class AsyncRequestCallbacks {
                          Tracing::Span& span) PURE;
 };
 
+class AsyncRequestCallbacks : public RawAsyncRequestCallbacks {
+public:
+  virtual ~AsyncRequestCallbacks() {}
+
+  void onSuccessRaw(Buffer::InstancePtr response, Tracing::Span& span) override;
```

Comment:
Comments? Why do we have the same callback class providing both raw and untyped? Can't one be implemented in terms of the other?

Reply:
If you subclass from AsyncRequestCallbacks then it overrides onSuccessRaw for you so that onSuccessUntyped is called as you are expected to override that.

I can make onSuccessRaw private in AsyncRequestCallbacks to make that clear and prevent accidental overrides of the wrong function.

Reply:
Updated to raw only API